frontend/templates/product-list/sections/HeroSection.tsx

```diff
@@ -140,7 +137,15 @@ function HeroTitle({
          data-testid="hero-title"
       >
          {children}
-         {page > 1 ? ` - Page ${page}` : ''}
+         {page > 1 ? (
+            <>
+               {' - Page'}
+               <span dangerouslySetInnerHTML={{ __html: '&nbsp;' }} />
```

Comment:
If we just don't want this to wrap at small sizes, can we just increase the width of the parent element?

Reply:
I'm sure we could, but I think the motivation to use a non-breaking space here is so that the 'Page' and page number are not split off into separate lines.